sys/include/net/gnrc/sixlowpan/ctx.h

```diff
@@ -23,7 +23,7 @@
  *
  * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
  */
-#ifndef GNRC_SIXLOWPAN_CTX_H_
+#if !(defined(GNRC_SIXLOWPAN_CTX_H_)) && defined(MODULE_GNRC_SIXLOWPAN_CTX)
```

Comment:
It messes with my ability to parse the header. Normally I ignore anything above `#define ..._H_`, since it normally isn't important content to me (except doc of course). Also what is the purpose of the new check anyway?